### PR TITLE
[PT] Update basic strings to version 1.1.1

### PR DESCRIPTION
**Pull request by brunoliveira8**

### CODE REVIEW

File: pt/lessons/basics/strings.md [hunk 1]
```diff
@@ -14,37 +16,38 @@ Strings em Elixir são nada mais que uma sequência de bytes. Vamos ver um exemp
...
-{104, 'ello'}
+`322` é o número Unicode de ł, representado em UTF-8 pelos dois bytes `197`, `130`.
 
-iex> Enum.reduce(char_list, "", fn char, acc -> acc <> to_string(char) <> "," end)
-"104,101,108,108,111,"
-```
+Ao programar em Elixir, geralmente usamos strings ao invés de listas de caracteres. O suporte a listas de caracteres é incluso principalmente por ser obrigatório para alguns módulos Erlang.
 
-Ao programar em Elixir, geralmente usamos Strings ao invés de listas de caracteres. O suporte a listas de caracteres é incluso principalmente por ser obrigatório para alguns módulos Erlang.
+Para mais informação, veja o oficial [`Getting Started Guide`](http://elixir-lang.org/getting-started/binaries-strings-and-char-lists.html).
```
Comment (pragmaticivan) [inline pt/lessons/basics/strings.md]: I think you need to translate `Getting Started Guide`

File: pt/lessons/basics/strings.md [hunk 2]
```diff
@@ -14,37 +16,38 @@ Strings em Elixir são nada mais que uma sequência de bytes. Vamos ver um exemp
...
-{104, 'ello'}
+`322` é o número Unicode de ł, representado em UTF-8 pelos dois bytes `197`, `130`.
 
-iex> Enum.reduce(char_list, "", fn char, acc -> acc <> to_string(char) <> "," end)
-"104,101,108,108,111,"
-```
+Ao programar em Elixir, geralmente usamos strings ao invés de listas de caracteres. O suporte a listas de caracteres é incluso principalmente por ser obrigatório para alguns módulos Erlang.
 
-Ao programar em Elixir, geralmente usamos Strings ao invés de listas de caracteres. O suporte a listas de caracteres é incluso principalmente por ser obrigatório para alguns módulos Erlang.
+Para mais informação, veja o oficial [`Guia de Introdução`](http://elixir-lang.org/getting-started/binaries-strings-and-char-lists.html).
```
Comment (anderkonzen) [inline pt/lessons/basics/strings.md]: It looks to me that `official` here is a bit misplaced. Maybe placing it at the end of the sentence? `... veja o Guia de Introdução oficial.`

File: pt/lessons/basics/strings.md [hunk 3]
```diff
@@ -129,7 +132,7 @@ end
 
 Primeiro vamos olhar para `anagrams?/2`. Estamos verificando se os parâmetros que recebemos são binários ou não. Essa é a forma de verificar se uma parâmetro é uma String em Elixir.
 
-Depois disso, estamos chamando a função que ordena as strings em ordem alfabética, primeiro deixando a string em letras minúsculas e então usando `String.graphemes`, que retorna a lista com os Graphemes da string. Bastante simples, não acha?
+Depois disso, chamamos a função que ordena as strings em ordem alfabética, primeiro deixamos a string com letras minúsculas e então usamos `String.graphemes`, que retorna a lista com os graphemes da string. Por fim, utilizamos `Enum.sort/1` para ordenar a lista. Bastante simples, não acha?
```
Comment (anderkonzen) [inline pt/lessons/basics/strings.md]: `String.graphemes' here is missing the arity.

File: pt/lessons/basics/strings.md [hunk 4]
```diff
@@ -1,6 +1,8 @@
 ---
-version: 0.9.1
+version: 1.1.1
 title: Strings
+redirect_from:
```
Comment (nscyclone) [inline pt/lessons/basics/strings.md]: Thanks, @brunoliveira8!
The `redirect_from` clause [should not be used](https://github.com/elixirschool/elixirschool/blob/master/README.md#translated-lesson) in translations and new lessons.

**Comment by brunoliveira8** [comment]
Thank you all for the feedback. I have updated the file.

**Comment by anderkonzen** [comment]
Thanks @brunoliveira8!

**Pull request merged**